acceptance/current/.acceptance/acceptance-cookbook/recipes/destroy.rb

```diff
@@ -0,0 +1,3 @@
+execute 'kitchen destroy' do
+  cwd File.join(File.dirname(__FILE__), "../../../../../../../..")
```

Comment:
Yes.  We have a ticket to create a `kitchen_suite` resource and optimizing all this ugliness is included.